docs/translations/it/missing-sites.mdx

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
This needs to be in the `frontend/content` directory alongside all the other translations of this page.

Reply:
So I have to create it in `frontend/content/it` and open another pull request?

Reply:
Yes but you don't need to open another PR, just change the location and make a new commit.